File: Dialogue/Dialogs.cpp
using namespace std;
#include <vector>
#include <string>
#include <sstream>
#include <DialogSet.h>
class Dialogs
{
private:
public:
    Displayable mainMenu(vector<string> mainMenu = {
        "[1]. Get new customer.",
        "[2]. Close tavern for the day.",
        "[3]. Close the game."
    });   
};

File: TavernSimCSharp/UiManagement/Dialogs.cs
public class Dialogs
{
    GameState gameState = GameState.Instance();
    public class mainMenu
    {
        public static Displayable headerMenuSelection = new Displayable("\u001bTavern Player UI:\u001b");
        public static Displayable menuSelection = new Displayable(
        lines:
            [
                "1: Get new customer in line",
                "2: Close tavern for the day",
                "3: End game",
            ]);


    } //Add rest of the dialogues
}